python/ray/tune/tests/test_trial_scheduler_pbt.py

```diff
@@ -32,6 +32,26 @@ def load_checkpoint(self, tmp_checkpoint_dir):
             self.a, self.b = pickle.load(fp)
 
 
+def MockTrainingFunc(config, checkpoint_dir=None):
+    iter = 0
+    a = config["a"]
+    b = config["b"]
+    # c = config["c"]
+
+    if checkpoint_dir:
+        checkpoint_path = os.path.join(checkpoint_dir, "model.mock")
+        with open(checkpoint_path, "rb") as fp:
+            a, b = pickle.load(fp)
+
```

Comment:
do you need to reset "iter"?

Reply:
I changed the test do this, but I don't think it actually matters for the test.